.github/workflows/docker-build.yml

```diff
@@ -0,0 +1,48 @@
+name: Docker Build
+
+on:
+  push:
+    branches: [ main ]
```

Comment:
This runs the test after merging in a PR or a direct push to `main` by maintainer.